backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/bayern/EhrenamtskarteBayern.kt

```diff
@@ -41,12 +43,15 @@ object EhrenamtskarteBayern : Pipeline {
             .addStep(SanitizeGeocode(config, logger, httpClient), logger) {
                 logger.info("== Sanitize data with geocoding ==")
             }
-            .addStep(PostSanitizeFilter(config, logger, httpClient), logger) {
+            .addStep(PostSanitizeFilter(config, logger, filteredStores), logger) {
                 logger.info("== Filter sanitized data ==")
             }
-            .addStep(FilterDuplicates(config, logger), logger) {
+            .addStep(FilterDuplicates(config, logger, filteredStores), logger) {
                 logger.info("== Filter duplicated data ==")
             }
+            .addStep(FilteredStoresCsvOutput(config, filteredStores), logger) {
+                logger.info("== Write CSV Filtered Stores")
```

Comment:
```suggestion
                logger.info("== Write CSV Filtered Stores ==")
```
nitpick to stick to the log format